packages/stryker/src/reporters/ProgressKeeper.ts

```diff
@@ -24,5 +26,23 @@ abstract class ProgressKeeper implements Reporter {
       this.progress.survived++;
     }
   }
+
+  protected getEtc() {
+    const timeLeft = Math.floor(this.timer.elapsedSeconds() / this.progress.tested * (this.progress.total - this.progress.tested));
```

Comment:
I think this should now be renamed to "totalSecondsLeft", as everything here has to do with "time left"